sys-fs/snapraid/snapraid-12.3-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="a backup program for disk array for home media centers"
+HOMEPAGE="https://www.snapraid.it/"
+SRC_URI="https://github.com/amadvance/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```

Comment:
This isn't controlling any test dependencies or conditional building of tests, so it can be removed (tests can be run even without this).